modules/MIPS/SQL/1_INSTALL/PQRS_clinical_rules.php

```diff
@@ -280,19 +277,25 @@
 ('PQRS_0455', 0, 0, 0, 0, '2017', '', 'patients:med', 'PQRS_0455', 1, 'X', 0),
 ('PQRS_0456', 0, 0, 0, 0, '2017', '', 'patients:med', 'PQRS_0456', 1, 'X', 0),
 ('PQRS_0457', 0, 0, 0, 0, '2017', '', 'patients:med', 'PQRS_0457', 1, 'X', 0),
+('PQRS_0459', 0, 0, 0, 0, '2017', '', 'patients:med', 'PQRS_0459', 1, 'X', 0),
```

Comment:
Shouldn't the PQRS codes stuff be in a table in the database versus hard coded php arrays?

Reply:
This is actually just an installer file for the database, not a hard-coded array.  If you install or upgraded the MIPS module, it wipes these out of the database and installs a fresh copy (along with all the other related tables).